### PR TITLE
Implement SelectCommand

### PR DESCRIPTION
**Pull request by tituswe**
Select functionality in progress

Selecting will allow users to highlight a client. Client's then information will be presented on the _Policy Panel_ of the GUI.

Implementing the boilerplate code for `SelectCommand` and `SelectCommandTest`.

For integration, let's keep the Select functionality native to the `UI` and `Logic` components. No further action required.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/SelectCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new SelectCommand object
+ */
+public class SelectCommandParser implements Parser<SelectCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
```
Comment (anunayajoshi) [inline src/main/java/seedu/address/logic/parser/SelectCommandParser.java:15]: I think the JavaDocs wasn't updated to SelectCommand from DeleteCommand here.

**Review by anunayajoshi — APPROVED** [review]
LGTM! Small change to JavaDocs needed.

File: src/main/java/seedu/address/logic/parser/SelectCommandParser.java (hunk 1 above)
Comment (tituswe) [reply src/main/java/seedu/address/logic/parser/SelectCommandParser.java:15]: Woops thank you! @anunayajoshi

**Review by pangrwa — APPROVED** [review]
LGTM

**Pull request merged**